simulator/src/data/load.js

```
@@ -1,7 +1,7 @@
 import { resetScopeList, newCircuit, switchCircuit } from '../circuit';
 import { setProjectName } from './save';
 import {
- scheduleUpdate, update, updateSimulationSet, updateCanvasSet, gridUpdateSet 
+ scheduleUpdate, update, updateSimulationSet, updateCanvasSet, gridUpdateSet
```

ESLint

[error] Expected indentation of 4 spaces but found 1.

(indent)